### PR TITLE
[NOJIRA] Add watcher

### PR DESCRIPTION
**Pull request by matthewdavidson**

### CODE REVIEW

File: watcher.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+const fs = require('fs');
+const ora = require('ora');
+const sass = require('node-sass');
+const chokidar = require('chokidar');
+const importer = require('node-sass-tilde-importer');
+const functions = require('bpk-mixins/sass-functions.js');
+
+const getCssFileName = name => name.replace(/\.scss/, '.css');
+
+const compileSass = file =>
```
Comment (matteo-hertel) [inline watcher.js]: How about Promisify for this? it'll be much neater

File: watcher.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      console.error(e);
+      console.error();
+    }
+  })
+  .on('unlink', async file => {
+    spinner.start(`Removed: ${file}`);
+    const cssFileName = getCssFileName(file);
+
+    try {
+      await unlinkFile(cssFileName);
```
Comment (k0nserv) [inline watcher.js:84]: Do we need to unlink the file? Hasn't that already been done by the OS at this point?

File: watcher.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      console.error(e);
+      console.error();
+    }
+  })
+  .on('change', async file => {
+    spinner.start(`Changed: ${file}`);
+    const cssFileName = getCssFileName(file);
+
+    try {
+      const result = await compileSass(file);
```
Comment (k0nserv) [inline watcher.js]: Can we extract the common code from these two events into a function?

File: watcher.js (hunk 2 above)
Comment (matthewdavidson) [reply watcher.js:84]: We are unlinking the `.css` file here, not the `.scss` file.

File: watcher.js (hunk 1 above)
Comment (matthewdavidson) [reply watcher.js]: Done! let me know what you think.

File: watcher.js (hunk 3 above)
Comment (matthewdavidson) [reply watcher.js]: Done - have a look!

**Review by k0nserv — APPROVED** [review]
Noice

**Pull request merged**